controllers/keda/suite_test.go

```diff
@@ -22,21 +22,19 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo"
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+	"github.com/kedacore/keda/v2/pkg/k8s"
+	"github.com/kedacore/keda/v2/pkg/scaling"
```

Comment:
There is a style rule for imports (and it's detected by static checks).
Imports have to be split in 3 groups, the first has to be go imports, the second one has to be the external packages and the last one has to be internal packages. 
Move this 3 imports to the end and split them in a group. Pleas, also join 
```
logf "sigs.k8s.io/controller-runtime/pkg/log"
	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```
with the second group